src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -613,17 +627,7 @@ void WaveTrackAffordanceControls::StartEditSelectedClipSpeed(
    if (!interval)
       return;
 
-   PitchAndSpeedDialog dlg(
-      ProjectAudioIO::Get(project).IsAudioActive(), *track, *interval,
-      &GetProjectFrame(project));
-
-   if (wxID_OK == dlg.ShowModal())
-   {
-      ProjectHistory::Get(project).PushState(
-         XO("Changed Pitch and Speed"), XO("Changed Pitch and Speed"),
-         UndoPush::CONSOLIDATE);
-      SelectInterval(project, *interval);
-   }
+   ShowClipPitchAndSpeedDialog(project, *track, *interval);
```

Comment:
I was mistaken -- I see it is used in more than one place, now that I fix CMakeLists.txt and rebuild.